src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const declarations = sourceString
+    .split(/;(?![^(]*\))/)
```

Comment:
The regular expression used in split might not handle all edge cases, such as when there are semicolons within strings or comments in the CSS. Consider refining it to handle these cases.

Reply:
done